### DISCUSSION TITLE
Rigblaster usage

### DISCUSSION THREAD

**Discussion by w8iss** [F4HTB/Universal_HamRadio_Remote_HTML5#27] [Q&A]
Wondering about usage of rigblaster and mfj usb soundcard interfaces with this html5 program.

Read the wiki page but didn't see any mention that they would work out of the box since they are usb interfaces. I would think they could work.

Figured I ask before I zap something. Already have zapped one ts-2000 with a usb to serial interface. Don't want to do that again.

**Comment by F4HTB** [comment]
Hello,

I hope you will have receive my response because I am not sure if the email was arrive.

If these interfaces are recognized as sound cards and serial interfaces, there should be no problems.

It will be necessary to check if the interfaces allowing to read and write in 8khz at the level of the sound driver.

You can use the alsacap utility.
https://www.f4htb.fr/2017/06/03/alsacap-bien-utils/

~$ alsacap
Card 31, ID `lp21', name `Loopback'
  Device 0, ID `Loopback PCM', name `Loopback PCM', 2 subdevices (2 available)
    1..32 channels, sampling rate 8000..768000 Hz
    Sample formats: S16_LE, S16_BE, S32_LE, S32_BE, FLOAT_LE, FLOAT_BE
      Subdevice 0, name `subdevice #0'
      Subdevice 1, name `subdevice #1'
  Device 1, ID `Loopback PCM', name `Loopback PCM', 2 subdevices (2 available)
    1..32 channels, sampling rate 8000..768000 Hz
    Sample formats: S16_LE, S16_BE, S32_LE, S32_BE, FLOAT_LE, FLOAT_BE
      Subdevice 0, name `subdevice #0'
      Subdevice 1, name `subdevice #1'
